experimental/dds/tree2/src/feature-libraries/editable-tree-2/proxies/proxies.ts

```diff
@@ -164,11 +167,14 @@ export function createObjectProxy<TSchema extends StructSchema, TTypes extends A
 				if (field !== undefined) {
 					return getProxyForField(field);
 				}
-				// TODO: Do this a better way.
-				if (key === treeNodeSym) {
-					return { schema };
+				switch (key) {
+					case treeNodeSym:
+						return content;
```

Comment:
@noencke - the 'content' object is the editable tree node (which has the schema).

I didn't want to mix it with the pending PRs, but renaming 'content & treeNode' -> 'editNode' would probably be clearer.